ansible/inventory/group_vars/apps.yaml

```diff
@@ -206,6 +206,14 @@ chaos_monkey:
   root_vol_size: 20
   tag_name: AEM Chaos Monkey
   include_stack: true
+  termination_settings:
+    calendar_open_hour: '9'
+    calendar_close_hour: '15'
+    calendar_timezone: 'Australia/Sydney'
+    scheduler_frequency: '5'
+    scheduler_frequency_unit: 'MINUTES'
```

Comment:
Hide the unit in stack provisioner data jinja template. Don't propagate to the user. User will get confused about syntax, is it MINUTES? is it MINUTE? is it minute?